### PR TITLE
Raise document errors

### PR DESCRIPTION
**Pull request by bidoubiwa**
related to #126

### CODE REVIEW

**Review by nicolasvienot — COMMENTED** [review]
While testing, I found a few issues :

- See comment about the notification `update failed` error causing an error and not popping up when using Meilisearch <= `0.23.1`
- When trying to update a collection that has no entry, I have the following error message, which does not seem accurate
Capture d’écran 2021-11-29 à 14 26 07

Also, you might want to update a few @param and @return in the comments on top of your functions in `connectors/meilisearch/index.js`

Other than that, this looks clean. Let me know if any of those issues were expected.

File: admin/src/components/Collections.js [hunk 1]
```diff
@@ -73,18 +70,39 @@ const Collections = ({ updateCredentials }) => {
...
       )
       if (response.error) errorNotifications(response)
 
-      setListenedCollection(prev => prev.filter(col => col !== collection))
-      setUpdatedCollections(false) // Ask for collections to be updated.
+      const { updateStatus } = response
+
+      updateStatus.map(update => {
+        if (update.status === 'failed') {
+          update.error.message = `Some documents could not be added: \n${update.error.message}`
```
[the remark this answers is not in the document]
Comment (nicolasvienot) [reply admin/src/components/Collections.js:94]: Btw, it's working fine with MeiliSearch `0.24.0`
Capture d’écran 2021-11-29 à 15 11 28

[the remark this answers is not in the document]
Comment (bidoubiwa) [reply admin/src/components/Collections.js:94]: Yes! This release is only compatible with v0.24 :) Thanks for raising the issue nonetheless

**Comment by bidoubiwa** [comment]
bors merge

**Review by nicolasvienot — APPROVED** [review]

**Pull request merged**